crates/libduckdb-sys/Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "libduckdb-sys"
-version = "0.10.2"
+version = "1.0.0"
```

Same comment,  could share the version from the root manifest.

```suggestion
version.workspace = true
```

---

When i tried this previously it didnt work due to the "publish crates" CI job not understanding workspace properties.

---

https://github.com/duckdb/duckdb-rs/actions/runs/9369017132/job/25793098113

I ended up updating the action anyway so it might be fixed now idk.

---

@Maxxen 
Might have been fixed: https://github.com/katyo/publish-crates/pull/669